README.md

```diff
@@ -54,13 +54,16 @@ Here’s a complete `package.json` example, for a hypothetical module named `@mo
     ".": "./src/moment.mjs",
     "./": "./src/util/",
     "./timezones/": "./data/timezones/",
-    "./timezones/utc": "./data/timezones/utc/index.mjs"
+    "./timezones/utc": "./data/timezones/utc/index.mjs",
+    "localdep": "./third-party/localdep.mjs"
   }
 }
 ```
 
 Within the `"exports"` object, the key string after the `'.'` is concatenated on the end of the name field, e.g. `import utc from '@momentjs/moment/timezones/utc'` is formed from `'@momentjs/moment'` + `'/timezones/utc'`. Note that this is string manipulation, not a file path: `"./timezones/utc"` is allowed, but just `"timezones/utc"` is not. The `.` is a placeholder representing the package name. The main entrypoint is therefore the dot string, `".": "./src/moment.mjs"`.
 
+For keys not beginning in `.`, these are mappings which apply internally to any imports made from within the package itself. So `src/moment.mjs` could contain for example `import dep from 'localdep'` which would load from a package-relative location. For example one could define `"moment": "./src/moment.mjs"` allowing the package to refer to itself by name as well in this way. These mappings cannot be used at all from outside of the package.
```

Comment:
This seems to preclude being able to remap a file to a node_module. Also what about paths beginning in `/`?

Reply:
Because this mapping operation happens early, it could even be a node_modules package lookup actually.

We could allow arbitrary paths or not. Thoughts?

Reply:
I actually think this should forbid paths that don't start with `.`.

`npm` 6.9 has an alias feature that can be used to remap bare imports - I don't think there's any benefit in us duplicating it.